modules/migration-adding-replication-repository-to-cam.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-adding-replication-repository-to-cam_{context}']
+= Adding a replication repository to the CAM web console
+
+.Procedure
+
+. In the *Replication repositories* section, click *Add replication repository*.
+
+. In the *Replication repository* window, fill in the following fields:
+
+* *Replication repository name*
+* *S3 bucket name*
+* *S3 bucket region*: Required for AWS S3 if the bucket region is not *us-east-1*. Optional for a generic S3 repository.
+* *S3 endpoint*: Required for a generic S3 repository. This is the URL of the S3 service, not the bucket, for example, `http://_<minio-gpte-minio.apps.cluster.com>_`.
```

Comment:
We will want to update this to reference an example NooBaa endpoint instead of minio

I am working on other updates to show an example of how to setup the S3 pre-reqs.
Will share later today.

Reply:
Changed URL to `http://s3-noobaa.apps.cluster.com`